aws/resource_aws_codebuild_project_test.go

```diff
@@ -985,6 +985,38 @@ func TestAccAWSCodeBuildProject_WindowsContainer(t *testing.T) {
 	})
 }
 
+func TestAccAWSCodeBuildProject_ARMContainer(t *testing.T) {
+	var project codebuild.Project
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_codebuild_project.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSCodeBuild(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCodeBuildProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCodeBuildProjectConfig_ARMContainer(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSCodeBuildProjectExists(resourceName, &project),
+					resource.TestCheckResourceAttr(resourceName, "environment.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "environment.2306861956.compute_type", "BUILD_GENERAL1_LARGE"),
```

Comment:
The `TypeSet` hash value (`2306861956`) copied here from the other test will likely not work since it will be different for the updated `compute_type` -- Please feel free to just remove these `TestCheckResourceAttr()` calls that include that value since its difficult to get this value in the current acceptance testing framework and Terraform will report any unexpected difference without these.